examples/sam/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "devDependencies": {
     "@types/aws-lambda": "^8.10.86",
+    "@types/jest": "^27.5.2",
```

Comment:
This change is needed because we are now running the tests outside of the `lerna` scope. For this reason the dependency (which is present in both the `examples/cdk` and the main package) is also needed here now.